llvmDebugInfoC/src/main/cpp/DebugInfoC.cpp

```diff
@@ -152,8 +146,10 @@ DICompositeTypeRef DICreateArrayType(DIBuilderRef refBuilder,
                                      uint64_t elementsCount) {
   auto builder = llvm::unwrap(refBuilder);
   auto range = std::vector<llvm::Metadata*>({llvm::dyn_cast<llvm::Metadata>(builder->getOrCreateSubrange(0, size))});
-  return llvm::wrap(builder->createArrayType(size, alignInBits, llvm::unwrap(refType),
-                                             builder->getOrCreateArray(range)));
+  auto type = builder->createArrayType(size, alignInBits, llvm::unwrap(refType),
+                                                           builder->getOrCreateArray(range));
+  builder->retainType(type);
```

Comment:
Is the same required for `DICreateMemberType`?

Reply:
No.